docs/FeatureManagement/Samples/TargetingFilter.tests.json

```diff
@@ -0,0 +1,211 @@
+[
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-DisabledDefaultRollout",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Aiden"},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Aiden is not part of the default rollout."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-EnabledDefaultRollout",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Blossom"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Blossom is part of the default rollout."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-TargetedUser",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Alice"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Alice is a targeted user."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-TargetedGroup",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Aiden", "groups":["Stage1"]},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Aiden now part of the 100% rolled out group Stage1 is now targeted."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-DisabledTargetedGroup",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"groups":["Stage2"]},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, no user Stage 2 is 50% rolloued out."
```

Comment:
How about something like: empty/no user will hit the 50% rollout of group stage 2, so it is targeted.